traffic_ops/traffic_ops_golang/perlconfig.go

```diff
@@ -136,6 +176,66 @@ func getPort(obj map[string]interface{}) (string, error) {
 	return strconv.Itoa(port), nil
 }
 
+func getProxyTimeout(obj map[string]interface{}) (int, error) {
```

Comment:
a lot of redundant code..  why not pass string index and reduce to a single func?

Reply:
Refactored.